tests/components/grid.test.ts

```diff
@@ -639,6 +642,45 @@ describe("Grid component", () => {
     });
   });
 
+  describe("filter icon vertical alignment", () => {
+    let icons: HTMLElement[];
+    let rowDims: HeaderDimensions;
+    let sheetId: UID;
+    beforeEach(async () => {
+      resizeRows(model, [1], DEFAULT_CELL_HEIGHT * 2);
+      createFilter(model, "B2");
+      await nextTick();
+      sheetId = model.getters.getActiveSheetId();
+      rowDims = model.getters.getRowDimensionsInViewport(sheetId, 1);
+      icons = Array.from(fixture.querySelectorAll<HTMLElement>(".o-filter-icon"));
+      expect(icons).toHaveLength(1);
+    });
+
+    test("filter icon vertical alignment top", async () => {
+      setStyle(model, "B2", { verticalAlign: "top" }, sheetId);
+      await nextTick();
+      const top = `${rowDims.start + FILTER_ICON_MARGIN + HEADER_HEIGHT}px`;
```

Comment:
you can just computed the value numerically and then retrieve the numerical value of the property with the helper `getStylePropertyInPx`  this can be applied to all the tests :)